doc/classes/TileSetScenesCollectionSource.xml

```diff
@@ -56,38 +56,38 @@
 			<return type="bool" />
 			<param index="0" name="id" type="int" />
 			<description>
-				Returns whether this TileSet source has a scene tile with id [code]id[/code].
+				Returns whether this TileSet source has a scene tile with id [param id].
 			</description>
 		</method>
 		<method name="remove_scene_tile">
 			<return type="void" />
 			<param index="0" name="id" type="int" />
 			<description>
-				Remove the scene tile with id [code]id[/code].
+				Remove the scene tile with id [param id].
 			</description>
 		</method>
 		<method name="set_scene_tile_display_placeholder">
 			<return type="void" />
 			<param index="0" name="id" type="int" />
 			<param index="1" name="display_placeholder" type="bool" />
 			<description>
-				Sets whether or not the scene tile with id [code]id[/code] should display a placeholder in the editor. This might be useful for scenes that are not visible.
+				Sets whether or not the scene tile with id [param id] should display a placeholder in the editor. This might be useful for scenes that are not visible.
```

Comment:
```suggestion
				Sets whether or not the scene tile with [param id] should display a placeholder in the editor. This might be useful for scenes that are not visible.
```